tests/Twig/Extension/MediaExtensionTest.php

```diff
@@ -133,6 +133,11 @@ public function getTemplate(): object
         return $this->template;
     }
 
+    /**
+     * @psalm-suppress InternalMethod
+     *
+     * @see https://github.com/twigphp/Twig/blob/3.x/src/TemplateWrapper.php#L24-L30
```

Comment:
We should create an issue/PR for this in twig.

Reply:
Why?, we are mocking a lot of internal twig things. I am not sure they will remove the internal here.

Reply:
It might be interesting to start a discussion in order to know/determine the right way to have a TemplateWrapper in the tests. I already remove some `@internal` annotation in Symfony, they seems opened to discussion.

Reply:
I don't think this is suposed to be tested as unit test. We should change this to be functional, but it is not the purpose of this PR.